DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,26 @@ public function getConfigTreeBuilder()
                     return $v;
                 })
             ->end()
+            ->beforeNormalization() // @deprecated toolbar in 1.3.0, use profiling instead
+                ->ifTrue(function ($v) {
+                    return is_array($v) && array_key_exists('toolbar', $v) && is_array($v['toolbar']);
+                })
+                ->then(function ($v) {
+                    if (array_key_exists('enabled', $v['toolbar']) && 'auto' === $v['toolbar']['enabled']) {
+                        $v['toolbar']['enabled'] = $this->debug; // @deprecated value auto in 1.3.0
+                    }
+
+                    if (array_key_exists('profiling', $v) && is_array($v['profiling'])) {
```

Comment:
imho we should consider configuration invalid if both profiling and toolbar exist. the user should decide for one.

do we have a problem at this stage with default values from the Configuration? i think they will only come after this step.